src/Types/RecursivePartial.ts

```diff
@@ -0,0 +1,5 @@
+export type RecursivePartial<T> = {
+    [P in keyof T]?: T[P] extends (infer U)[]
+        ? RecursivePartial<U>[]
+        : T[P] extends object ? RecursivePartial<T[P]> : T[P]
+};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)